modules/holodeckb2b-core/src/main/java/org/holodeckb2b/security/handlers/CreateWSSHeaders.java

```diff
@@ -285,8 +288,12 @@ private void setupSignature(final MessageContext mc, final ISigningConfiguration
      */
     private void setupEncryption(final MessageContext mc, final IEncryptionConfiguration encCfg,
                                  final PasswordCallbackHandler pwdCBHandler) {
+        System.out.println("[setupEncryption]>");
```

Comment:
Fixed. Sorry for that. That was temporary souts, which I use for writing tests. They are no longer needed.